components/List/List.js

```diff
@@ -38,7 +38,7 @@ export default function List({
 
   return (
     <div>
```

Comment:
Better use a Fragment `<>` instead of an `<div>`.

Reply:
i did :)